sycl/test/extensions/annotated_usm/annotated_usm.cpp

```diff
@@ -0,0 +1,243 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -fsyntax-only -Xclang -verify -Xclang -verify-ignore-unexpected=note %s
+// expected-no-diagnostics
+
+// Common tests for annotated USM allocation
```

Comment:
Can you make the description more precise, e.g., "Compile-time tests for annotated USM allocation functions".
Also, the other test file (under test-e2e) should be commented as "Runtime tests for annotated USM allocation functions."